public/app/app.ts

```diff
@@ -81,7 +82,7 @@ export class GrafanaApp {
     initExtensions();
     configureStore();
 
-    standardEditorsRegistry.setInit(getStandardOptionEditors);
+    standardEditorsRegistry.setInit(getAllOptionEditors);
```

Comment:
this registers editors that require runtime -- I think that is good, and we should potentially move some of the existing `@grafana/ui` options to core